sites/cheerpx/src/content/docs/11-guides/input-output.md

```diff
@@ -0,0 +1,49 @@
+---
+title: input and output CheerpX
+description: Summarization on how to get data in and out of CheerpX VM
+---
+
+## CheerpX console input and output
+
+CheerpX provides two options for handling console input and output: the `built-in` console and a `custom` console. The purpose of these consoles are to interact with the running application, execute commands and, view output. The custom console allows you to create an interface that fits the specific needs of your application.
```

Comment:
Prefer _italics_ to `code` for built-in and custom. We should only use the `code` markers for names which corresponds to APIs